Packs/CommonScripts/ReleaseNotes/1_19_97.md

```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+
+##### GetServerURL
+- Updated the Docker image to: *demisto/python3:3.12.11.4417419*.
+
+- Updated the GetServerURL script to use return_results.
```

Comment:
```suggestion
- Updated the **GetServerURL** script to use ***return_results*** instead of deprecated ***return_outputs***.
```
Please also ask for docs review